OpenSim/Simulation/Model/TwoFrameLinker.h

```diff
@@ -566,6 +585,34 @@ template <class C, class F>
 void TwoFrameLinker<C, F>::addInPhysicalForcesFromInternal(
         const SimTK::State& s,
         SimTK::Vec6 f, SimTK::Vector_<SimTK::SpatialVec>& physicalForces) const
+{
+    // A `ForceConsumer` that applies body spatial vectors from `producePhysicalForcesFromInternal`
```

Comment:
This adaptor was written so that the maths around `TwoFrameLinker` is all in one place (the `ForceConsumer`-based one), rather than having two almost-identical implementations of the maths.